apps/dashboard/src/app/(app)/team/[team_slug]/[project_slug]/(sidebar)/webhooks/components/webhook-configs-table.tsx

```diff
@@ -0,0 +1,406 @@
+                            onClick={() => {
+                              setDeletingWebhook(config);
+                            }}
+                          >
+                            <TrashIcon className="mr-2 h-4 w-4" />
+                            Delete
+                          </DropdownMenuItem>
+                        </DropdownMenuContent>
+                      </DropdownMenu>
+                    </TableCell>
+                  </TableRow>
+                ))}
+              </TableBody>
+            </Table>
+          </TableContainer>
+
+          {showPagination && (
+            <div className="rounded-b-lg border border-t-0 bg-card p-4">
+              <PaginationButtons
+                activePage={page}
+                onPageClick={setPage}
+                totalPages={totalPages}
+              />
+            </div>
+          )}
+        </div>
+      )}
+
+      <CreateWebhookConfigModal
+        onOpenChange={setIsCreateModalOpen}
+        open={isCreateModalOpen}
+        projectSlug={props.projectSlug}
+        teamSlug={props.teamSlug}
+        topics={props.topics}
+      />
+
+      {editingWebhook && (
+        <EditWebhookConfigModal
+          onOpenChange={(open) => {
+            if (!open) setEditingWebhook(null);
+          }}
+          open={!!editingWebhook}
+          projectSlug={props.projectSlug}
+          teamSlug={props.teamSlug}
+          topics={props.topics}
+          webhookConfig={editingWebhook}
+        />
+      )}
+
+      <DeleteWebhookModal
+        isPending={deleteMutation.isPending}
+        onConfirm={() => {
+          if (deletingWebhook) {
+            deleteMutation.mutate(deletingWebhook.id);
+          }
+        }}
+        onOpenChange={(open) => {
+          if (!open) setDeletingWebhook(null);
+        }}
+        open={!!deletingWebhook}
+        projectSlug={props.projectSlug}
+        teamSlug={props.teamSlug}
+        webhookConfig={deletingWebhook}
+      />
+    </div>
+  );
+}
+
+const sortByIcon: Record<SortById, React.FC<{ className?: string }>> = {
+  createdAt: CalendarIcon,
+  description: LetterTextIcon,
+  destinationUrl: WebhookIcon,
+  pausedAt: CheckIcon,
+};
+
+function SortDropdown(props: {
+  sortBy: SortById;
+  onSortChange: (value: SortById) => void;
+  hasActiveFilters: boolean;
+}) {
+  const values: SortById[] = [
+    "description",
+    "createdAt",
+    "destinationUrl",
+    "pausedAt",
+  ];
+  const valueToLabel: Record<SortById, string> = {
+    createdAt: "Creation Date",
+    description: "Description",
+    destinationUrl: "Destination URL",
+    pausedAt: "Requests",
+  };
+
+  return (
+    <DropdownMenu>
+      <DropdownMenuTrigger asChild>
+        <Button className="gap-1.5 rounded-full bg-card" variant="outline">
+          <ArrowUpDownIcon className="size-4 text-muted-foreground" />
+          <span className="hidden lg:inline">Sort by</span>
```

Comment:
remove `hidden lg:inline` here - there's enough space to show "Sort by" on mobile viewport: 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/dec2cc09-1173-4f9e-9ba5-e49803fd6f7b.png)